### ISSUE TITLE
build release images for i7 model too

### ISSUE THREAD

**Issue by merge** [merge/coreboot-x230#2]
Basically we need to have the microcode version information. It should be tested well before releasing.

**Comment by merge** [comment]
New strategy: Make sure the most recent Intel microcode updates are in coreboot upstream. For every CPU model.

We simply release when there's a new one in coreboot.

**Comment by mikeryan** [comment]
Do you know when the microcode update supporting i7 will make it into coreboot upstream?

**Comment by merge** [comment]
That will be just a few days after Intel releases it. We collect updates while we wait here: https://github.com/merge/coreboot-x230/issues/3

If we have indication that it'll take longer, I might do a release in the meantime, including the 20171117 microcode package, because for the X230, even that includes a newer microcode version than coreboot master now...

But coreboot upstream seems to wait for Intel now.

**Comment by merge** [comment]
I did a release that includes the most recent microcode revision now. coreboot upstream still includes revision `1b` and the 0.0.3 image includes revision `1c` which is part of the latest release by Intel 20171117 (and was also in 20180108).

You cannot trivially rebuild that, as I added it to my coreboot tree only, but as soon as Intel finally does a new release, we'll help pushing it into coreboot and we're back on track. And I have the impression that we'll get a new revision for the X230 with that anyways.
